docs/ref/forms/fields.txt

```diff
@@ -506,20 +506,30 @@ For each field, we describe the default widget used if you don't specify
         A list of formats used to attempt to convert a string to a valid
         ``datetime.datetime`` object.
 
-    If no ``input_formats`` argument is provided, the default input formats are::
-
-        ['%Y-%m-%d %H:%M:%S',    # '2006-10-25 14:30:59'
-         '%Y-%m-%d %H:%M',       # '2006-10-25 14:30'
-         '%Y-%m-%d',             # '2006-10-25'
-         '%m/%d/%Y %H:%M:%S',    # '10/25/2006 14:30:59'
-         '%m/%d/%Y %H:%M',       # '10/25/2006 14:30'
-         '%m/%d/%Y',             # '10/25/2006'
-         '%m/%d/%y %H:%M:%S',    # '10/25/06 14:30:59'
-         '%m/%d/%y %H:%M',       # '10/25/06 14:30'
-         '%m/%d/%y']             # '10/25/06'
+    The field always accept strings in ISO 8601 formatted dates or similar
+    recognized by :func:`django.utils.dateparse.parse_datetime`. Some examples
```

Comment:
So, question, are we sure about _always_? (i.e. Do we want to allow disabling? — Perhaps we can do that IF someone actually wants it...)

Need as `s` on `accepts`. 

Can I suggest an addition to the `attribute` description above too: 

```
A list of formats used to attempt to convert a string to a valid 
``datetime.datetime`` object, in addition to ISO 8601 formats. 
```

Not 100% happy about how that reads, which might be a pointer towards the extra API... (???)

Reply:
Ok, me neither really. Maybe one appears but... DEAL_WITH_LATER.